src/modules/identity/services/team-user.service.ts

```diff
@@ -461,6 +480,18 @@ export class TeamUserService {
     return response;
   }
 
+  async OwnerDetails(
```

Comment:
change name to ownerDetails or rename it something more understandable like getOwnerDetails